remoting/src/test/java/org/springframework/security/remoting/dns/JndiDnsResolverTests.java

```diff
@@ -92,6 +92,16 @@ public void testResolveServiceIpAddress() throws Exception {
 		assertThat(ipAddress).isEqualTo("63.246.7.80");
 	}
 
+	@Test
+	public void resolveServiceIpAddressWithPort() throws Exception {
```

Comment:
NB - considered prefixing this with "test" to match format of rest of file, but after checking the guidelines it specifically says new tests should not prefix "test" so just let me know if I misread that.

Reply:
You are correct not to prefix the test with "test". 
As we add more tests, we generally follow the format `methodWhenConditionThenResult`.
This one could be named something like `resolveServiceIpAddressAndPortWhenExistsThenReturnsIpAdressAndPort`.